cra-tf-validate-ignore-rules.json

```diff
@@ -11,12 +11,6 @@
             "description": "Check whether Flow Logs for VPC are enabled",
             "ignore_reason": "Rule is related to VPC, not OCP Clusters",
```

Comment:
@ocofaigh all these rules can be removed after this is merged, just incase I forget...